spec/aruba/api/filesystem_spec.rb

```diff
@@ -4,6 +4,23 @@
 RSpec.describe Aruba::Api::Filesystem do
   include_context "uses aruba API"
 
+  describe "#append_lines_to_file" do
+    let(:path) { @file_path }
```

Comment:
pop these as top level? Seem to be used in nearly all of the examples (I scrolled down past L40)

Reply:
That makes sense, yes.